### DISCUSSION TITLE
⭐ Xcode 15.3 release

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2357] [Announcements]
A new build image with Xcode 15.3 Beta 1 is now available. This image comes with macOS Sonoma 14.3 and has all the latest SDKs and runtimes including visionOS installed. You can use Xcode 15.3 by specifying `15.3` or `edge` as the Xcode version in your build configuration.

See more details about the build machine specs and preinstalled software in our [documentation](https://docs.codemagic.io/specs/versions-macos/).

**Comment by mikhail-tokarev** [comment]
> Please note that as of April 2024 all iOS and iPadOS apps submitted to the App Store must be built with a minimum of Xcode 15 and the iOS 17 SDK.

https://developer.apple.com/ios/submit/#:~:text=Please%20note%20that,iOS%2017%20SDK

**Comment by helinanever** [comment]
The build image has been updated with Xcode 15.3 Beta 2.

**Comment by helinanever** [comment]
Xcode 15.3 Beta 3 is now available.

**Comment by helinanever** [comment]
We have 15.3 RC 1 available now.

**Reply by vince1393** [reply]
Looks like there was an RC2 that came out yesterday along with the official 15.3 release version today. Will that be available soon?

**Reply by helinanever** [reply]
The build image has just been updated to the 15.3 release. 🎉

**Comment by helinanever** [comment]
Xcode 15.3 release version is already available on macOS M1 and M2 machines.
